alchemy-json_api.gemspec

```
@@ -19,7 +19,7 @@ Gem::Specification.new do |spec|
   spec.files = Dir["{app,config,db,lib}/**/*", "LICENSE", "Rakefile", "README.md"]
 
   spec.add_dependency "alchemy_cms", [">= 7.0.0.a", "< 8"]
-  spec.add_dependency "jsonapi.rb", [">= 1.6.0", "< 2.1"]
+  spec.add_dependency "jsonapi.rb", [">= 1.6.0", "<= 2.1.1"]
```

Could we change this to `< 2.2`? I think patch level releases should be fine.

```suggestion
  spec.add_dependency "jsonapi.rb", [">= 1.6.0", "< 2.2"]
```

---

@manleyac made the changes